content/agent/exporters/zipkin/_index.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Zipkin"
+date: 2019-02-13T22:23:58-08:00
+logo: /img/zipkin-logo.jpg
+---
+
+- [Introduction](#introduction)
+- [Configuration](#configuration)
+    - [Format](#format)
+    - [Example](#example)
+- [End to end example](#end-to-end-example)
+    - [Running ocagent](#running-ocagent)
+    - [Running Application Code](#running-application-code)
+- [Results](#results)
+    - [All traces](#all-traces)
+    - [Single trace](#single-trace)
+    - [Trace detail](#trace-detail)
+- [References](#references)
+
+
+### Introduction
+
+ocagent allows one to export traces to Zipkin
+![](/images/ocagent-exporter-zipkin-all-schematics.png)
+
+### Configuration
+
+In the ocagent's YAML configuration file, under section "exporters" and sub-section "zipkin" configure fields: 
+
+#### Format
```

Comment:
nit: You might also want to mention the "self-loop" issue with Zipkin (receiver and exporter both point to the same address).

Reply:
That's a great idea! Currently ocagent will fail and warn the user of the self loop, but sure, I've added a warning, thanks!